corpus/colorscheme_statement.txt

```diff
@@ -0,0 +1,34 @@
+================================================================================
+Colorscheme alone
+================================================================================
+
+colorscheme
+
+--------------------------------------------------------------------------------
+
+(script_file
+  (colorscheme_statement))
+
+================================================================================
+Colorscheme named
+================================================================================
+
+colorscheme kanagawa
+
+--------------------------------------------------------------------------------
+
+(script_file
+  (colorscheme_statement
+    (name)))
+
+================================================================================
+Colorscheme weird
```

Comment:
Lack of inspiration for this one 😅

Reply:
Lol that's no problem, it will draw attention when needed hehe